applications/shards/game/game.soy

```diff
@@ -0,0 +1,93 @@
+{namespace tictactoe}{template .game}
+{@param username: string | null}
+<div>
+    <a href="/login">
+        <button class="logbutton" onclick="sendLogout()">Logout</button>
+    </a>
+    <h4>{$username}</h4>
+</div>
+<div class="header">
+    <h1>Tic Tac Toe</h1>
+    <h3>Do you want to be "X" or "O"?</h3>
+    <button onclick="setX()" id="x">X</button>
+    <button onclick="setO()" id="o">O</button>
+    <button onclick="newGame()">New Game</button>
+</div>
+</div>
+    <div id="tableplace">
+       /*  <table id="gameBoard">
+            <tr>
+                <td>1</td>
+                <td>2</td>
+                <td>3</td>
+            </tr>
+            <tr>
+                <td>1</td>
+                <td>2</td>
+                <td>3</td>
+            </tr>
+            <tr>
+                <td>1</td>
+                <td>2</td>
+                <td>3</td>
+            </tr>
+        </table> */
+    </div>
+
+    <script type="text/javascript">
+        var playerOne = "";
+
+        function setX(){lb}
+            playerOne = "x";
+        {rb}
+
+        function setO(){lb}
+            playerOne = "o";
+        {rb}
+
+        function newGame(){lb}
+            var tbl = document.getElementById("gameboard");
```

Comment:
Consider using let/const instead of var everywhere since let/const is the current standard.

See [this guidance](https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75#:~:text=%60const%60%20is%20a%20signal%20that,always%20the%20entire%20containing%20function.).